KProcessHacker/dynimp.c

```diff
@@ -37,11 +37,8 @@ VOID KphDynamicImport(
 {
     PAGED_CODE();
 
-    KphDynPsGetProcessProtection = (PPS_GET_PROCESS_PROTECTION)(
-                       KphGetSystemRoutineAddress(L"PsGetProcessProtection"));
-
-    KphDynRtlImageNtHeaderEx = (PRTL_IMAGE_NT_HEADER_EX)(
-                           KphGetSystemRoutineAddress(L"RtlImageNtHeaderEx"));
+    KphDynPsGetProcessProtection = (PPS_GET_PROCESS_PROTECTION)KphGetSystemRoutineAddress(L"PsGetProcessProtection");
```

Comment:
In general I try to keep under 80 character width, no more that 100, for the sake of viewing the code. Formatting changes feel odd to go into a commit titled "Update KPH for VS2022".